cms/src/test/resources/test-data/api-specifications/SwaggerCodeGenOpenApiSpecificationJsonToHtmlConverterTest/openapi-v3-spec-empty-req-sec.json

```diff
@@ -0,0 +1,5263 @@
+{
+    "openapi": "3.0.0",
+    "info": {
```

Comment:
This file - as well as the corresponding HTML one - still looks way too big.

See comment to file [openapi-v3-spec-empty-req-sec.json](https://github.com/NHS-digital-website/hippo/pull/1076/files#diff-0b473cb70a9920aa524c94df9f7d7f97609d6bafb6ee5c73251e88f053927bd2).

Reply:
Fixed

Reply:
Fixed